ckanext/external_storage/tests/__init__.py

```diff
@@ -11,15 +11,25 @@ class FunctionalTestBase(helpers.FunctionalTestBase):
     _load_plugins = ['external_storage']
 
 
+def get_context(user):
+    userobj = core_model.User.get(user['name'])
+    return {
+        'model': core_model,
+        'user': user['name'],
+        "auth_user_obj": userobj,
+        'ignore_auth': False
+    }
+
+
 @contextmanager
 def user_context(user):
     # type: (Dict[str, Any]) -> Dict[str, Any]
     """Context manager that creates a CKAN context dict for a user, then
     both patches our `get_user_context` function to return it and also
     yields the context for use inside tests
     """
-    userobj = model.User.get(user['name'])
-    context = {"model": model,
+    userobj = core_model.User.get(user['name'])
```

Comment:
DRY. If you wrote `get_context`, factor out the identical code from here and just call `get_context`. But really, I'm not sure you need this, because you can probably just use `user_context(user)` wherever you need `get_context()` in your tests.